src/main/drivers/timer_hal.c

```diff
@@ -807,69 +819,90 @@ _TIM_IRQ_HANDLER(TIM1_UP_TIM16_IRQHandler, 16);    // only timer16 is used, not
 _TIM_IRQ_HANDLER(TIM1_TRG_COM_TIM17_IRQHandler, 17);
 #endif
 
-void timerInit(void)
+#ifdef USE_TIMER_MGMT
+TIM_TypeDef* getTimByTag(const timerTag_t tag)
```

Comment:
(you can use tricks from `io_def.h` to replace this with table)

Reply:
I'll leave the tricks to the more experienced amongst us. I am personally not keen on all the obfuscation. It makes it a pain to debug and understand what is going on. Sometimes its better to take the simplest option (at the cost of performance) for human readability. We aren't all programming gods, and we want contributions from as many as possible (i.e. it is not the elegance of your code that matters, but the elegance of the desired product outcome aka feature).